dev/com.ibm.ws.transport.http/src/com/ibm/ws/genericbnf/internal/BNFHeadersImpl.java

```diff
@@ -927,6 +941,9 @@ public List<HeaderField> getHeaders(HeaderKeys key) {
             throw new IllegalArgumentException("Null input provided");
         }
         List<HeaderField> list = new ArrayList<HeaderField>();
+        if (headerKeysByName.get(key.getName()) == null) {
+            return list;
+        }
```

Comment:
Again lose of the debug statement:

```suggestion
        if (headerKeysByName.get(key.getName()) != null) {
            HeaderElement elem = findHeader(key);
            while (null != elem) {
                if (!elem.wasRemoved()) {
                    list.add(elem);
                }
                elem = elem.nextInstance;
            }
        }
```

Reply:
Good catch, thanks